cmake/assets/protos.cmake

```diff
@@ -0,0 +1,9 @@
+message(STATUS "FetchContent: generated protos for Windows")
+
+FetchContent_Declare(
+    protos
+    URL https://github.com/jlknauer/cpp-sc2/releases/download/v1.4.1/generated-s2clientprotocol-db14236-protobuf-1a74ba4.zip
```

Comment:
Please notify me when we have final version of the bindings for this PR and I'll upload to the S3 store.